stlTranslator/SourceFiles/read.cpp

```diff
@@ -22,6 +22,7 @@ void Read::read() {
             }
```

Comment:
File name is hardcoded, each time you have to change the file, you have to come up here and change the name, which can easily be done if it was inputted in main function. If you have to input user provided file then this would be disastrous.

Reply:
Done.